boards/arm/sensortile_box_pro/sensortile_box_pro.dts

```diff
@@ -180,6 +180,16 @@
 
 		drdy-pin = <2>;
 	};
+
+	lis2du12: lis2du12@1 {
+		compatible = "st,lis2du12";
+		spi-max-frequency = <DT_FREQ_M(10)>; /* 10 MHz */
```

Comment:
drop the comment, the code is meaningful enough without it, it's just going to be copy pasted around and go out of date

Reply:
ok, makes sense